SPECS/binutils/binutils.spec

```diff
@@ -119,6 +120,9 @@ make %{?_smp_mflags} check
 %{_libdir}/libctf.so.0.*
 
 %changelog
+* Thu Feb 09 2023 Dan Streetman <ddstreet@microsoft.com> 2.36.1-3
+- CVE-2022-4285
```

Comment:
Nit: Elaborate that you created a patch. Sometimes we create nopatches, do upgrades etc.